tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -595,14 +598,20 @@ def create_pool(self, **kw):
             self.exec_cmd(cmd="rbd pool init {}".format(kw.get("poolname")))
 
     def create_image(self, **kw):
-        cmd = "rbd create -s {} {} --image-feature exclusive-lock,journaling".format(
```

Comment:
[Optional] could you add doc string for this as well.

Reply:
done.